examples/abitest/config/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+readonly MODULES_DIR="${DIR}/../../target/wasm32-unknown-unknown/release"
+readonly OUT_DIR="${DIR}/../bin"
+mkdir --parents "${OUT_DIR}"
+
+bazel run //oak/common:app_config_serializer -- \
```

Comment:
So this PR doesn't remove Bazel for building examples.
Why was it moved to `build.sh`?

Reply:
It's about the linked issue, which is removing Bazel from the invocation of the app_serializer. It is described in the first comment, but not reflected in the title, so I will change that. I think we will always build examples with Bazel (c++ ones), we don't have a goal to remove that.